Packs/GoogleVault/Integrations/GoogleVault/GoogleVault.py

```diff
@@ -1509,6 +1511,29 @@ def main():
         return_error(str(e))
 
 
+def process_main():
+    """setup stdin to fd=0 so we can read from the server"""
+    sys.stdin = os.fdopen(0, "r")
+    sub_main()
+
+
+def main():
+    # When running big queries, like 'gvault-get-mail' the memory might not freed by the garbage
```

Comment:
is that the only relevant command? if so why not open a new process just in case of that command?